src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -219,6 +221,26 @@ void Engine::ApplyPerformanceHints(std::map<std::string, std::string> &config, c
         return std::pair<std::string, Engine::StreamCfg>(std::to_string(streams_info.num_streams), streams_info);
     };
 
+    auto getNumStreams = [&](int nstreams) {
+        StreamCfg streams_info;
+        int model_prefer = GetModelPreferThreads(ngraphFunc);
+        const std::vector<std::vector<int>> proc_type_table = getNumOfAvailableCPUCores();
+        const std::vector<std::vector<int>> stream_info_table =
+            get_streams_info_table(nstreams, engConfig.streamExecutorConfig._threads, model_prefer, proc_type_table);
+        streams_info = ParseStreamsTable(stream_info_table);
+
+        auto num_requests = config.find(CONFIG_KEY(PERFORMANCE_HINT_NUM_REQUESTS));
+        if (num_requests != config.end()) {  // arrived with config to the LoadNetwork (and thus higher pri)
```

Comment:
`num_requests` should be changed before `get_streams_info_table()`. If `num_requests` is small than `nstreams`, `nstreams` should be updated to `num_requests`.

Reply:
You are right. I moved it to before get_streams_info_table().